packages/contracts/contracts/Passport.sol

```diff
@@ -43,6 +46,34 @@ contract Passport is ERC721, Ownable {
     emit Purchase(msg.sender, price, _id, tokenURI(_id));
   }
 
+  function transfer(
+      address from,
+      address to,
+      uint256 tokenId
+  ) external payable {
+    enableTransfer = true;
+    if (royalty > 0) {
```

Comment:
is royalty a flat value rather than a percentage of `msg.value`? Also seems like the owner of the NFT is transferring ownership for free?

Reply:
We can change the royalty it to a percentage.

Hmmm....are you suggesting that the seller pay the royalty rather than the buyer?

Reply:
Let's say someone sells a passport for 100 ETH on our marketplace and royalty is set to 5%. I assume we want to make 3 txns:
5ETH to owner
2.5ETH to CabinDAO
92.5ETH to the seller.
right?

Reply:
right, but how it's currently set up, someone is selling the NFT for free because it doesn't use `msg.value` at all. and instead is using `royalty` as a flat value